nasa-gibs/worldview
# WV-3382: Add 6 AMSR2 L3 weekly and monthly layers

## Description

Fixes #WV-3382 .

Added 6 Weekly and Monthly Level 3 AMSR2 layers

- Total Precipitable Water
- Cloud Liquid Water Path
- Ocean Wind Speed

## How To Test

- `git checkout wv-3382-add-amsr2-l3-weekly-monthly-layers`
- `npm run build && npm start`
- Open with [these URL parameters](http://localhost:3000/?l=Reference_Labels_15m(hidden),Reference_Features_15m(hidden),Coastlines_15m,AMSRU2_L3_Total_Precipitable_Water_Monthly,AMSRU2_L3_Total_Precipitable_Water_Weekly,AMSRU2_L3_Ocean_Wind_Speed_Monthly,AMSRU2_L3_Ocean_Wind_Speed_Weekly,AMSRU2_L3_Ocean_Wind_Speed_Daily,AMSRU2_L3_Cloud_Liquid_Water_Monthly,AMSRU2_L3_Cloud_Liquid_Water_Weekly,VIIRS_NOAA21_CorrectedReflectance_TrueColor(hidden),VIIRS_NOAA20_CorrectedReflectance_TrueColor(hidden),VIIRS_SNPP_CorrectedReflectance_TrueColor(hidden),MODIS_Aqua_CorrectedReflectance_TrueColor(hidden),MODIS_Terra_CorrectedReflectance_TrueColor&lg=true&t=2024-10-25-T18%3A16%3A25Z)
- Check to make sure the layers display, you can take an image snapshot, layer descriptions are accurate.


## PR Submission Checklist

This is simply a reminder of what we are going to look for before merging your code.

- I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/main/.github/CONTRIBUTING.md) doc
- I have added necessary documentation (if applicable)
- I have added tests that prove my fix is effective or that my feature works (if applicable)
- Any dependent changes have been merged and published in downstream modules (if applicable)

## Merging

Please use the `squash and merge` commit method unless each commit in your branch is vital to the commit history of main.

@nasa-gibs/worldview
